src/lib.rs

```diff
@@ -10,6 +10,7 @@ mod keyfile;
 pub(crate) mod variant_dictionary;
 pub(crate) mod xml_parse;
 
+pub mod otp;
```

Comment:
@DavidVentura why `pub`? Couldn't it be `pub(crate)` or just `mod`?

Reply:
If it's because of the tests, I would suggest moving the tests inside the `otp.rs` module, so that we don't have to expose the internal API only for testing.

Reply:
I think it was because I needed access from the Python bindings, but I'm not sure -- regardless, it's not relevant for this change, I'll remove it